Tests/Approval.Tests/ApprovedApi/FluentAssertions/net47.verified.txt

```diff
@@ -1665,6 +1665,10 @@ namespace FluentAssertions.Formatting
 }
 namespace FluentAssertions.Numeric
 {
+    public class ByteAssertions : FluentAssertions.Numeric.NumericAssertions<byte>
```

Comment:
Since we still return `NumericAssertions<byte>` in `AssertionExtensions.cs` can we make these classes `internal`?